pkgs/development/compilers/gambit/build.nix

```diff
@@ -13,29 +15,35 @@
 # However, -Os seems more consistent in winning slightly against both -O1 and -O2,
 # and is overall 15% faster than -O2. As for compile times, -O1 is fastest,
 # -Os is about 29%-33% slower than -O1, while -O2 is about 40%-50% slower than -O1.
-# Overall, -Os seems like the best choice, and that's what we now use.
+#
+# Overall, -Os seems like the best choice, but I care more about compile-time,
+# so I stick with -O1, which is also the default for Gambit.
 
 stdenv.mkDerivation rec {
-  pname = "gambit";
-  inherit version;
-  inherit src;
 
+  optimizationSetting = "-O1";
```

Comment:
Wouldn't it be better in the `let` above `mkDerivation`

Reply:
The ultimate goal is to make it configurable in an overlay. Should it be moved to `pkgs.gambit-support.optimizationSetting` or something?

Reply:
Maybe make it an argument of the function then? Right now it is defined in a way that is very hard to override, as `configureFlags` use _Nix_ variable expansion, not built-time shell parameter expansion.

Reply:
As in `gambitOptimizationSetting ? "-O1"` in the top parameters?

Reply:
Yes.